.eslintrc

```diff
@@ -1,7 +1,9 @@
 {
   "parserOptions": {
-    "ecmaVersion": 8
+    "ecmaVersion": 8,
+    "sourceType": "module"
   },
+  "parser": "typescript-eslint-parser",
```

Comment:
Eslint is only used for js code, so I don't think there is a need to specify parser here.